SystemInfo/Collector/DoctrineDatabaseSystemInfoCollector.php

```diff
@@ -33,7 +33,7 @@ public function __construct(Connection $db)
      *  - type
      *  - name
      *  - host
-     *  - username
+     *  - username.
```

Comment:
This one is strange, but ok 🙃

Reply:
Apparently lists are treated as a sentence that requires dot at the end. From spelling POV (at least in Polish) it makes perfect sense ;)